doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+### Financial chart / Stock Chart 
+
+The $PlatformShort$ Financial or Stock Chart, is a composite visualization that renders stock data and financial data in a time-series chart that includes interactive visual elements in a toolbar like day / week / month filters, chart type selection, volume type selection, indicators selection and trends lines selection. Designed for customization, the $PlatformShort$ Stock Chart can be customized in any way to give an easier visualization and interpretation of your data. The financial chart renders the date-time data along the X-Axis (bottom labels) and shows fields like Open, High, Low and Close volumes. The type of chart to render the Time-Series data can be Bar, Candle, Column, or Line. Learn more about our [financial charts](chart-types-financial.md).
+
+### Column chart 
+
+The $PlatformShort$ Column Chart, or Column Graph is among the most common category chart types used to quickly compare frequency, count, total, or average of data in different categories with data encoded by vertical bars of equal width and differing lengths. They are ideal for showing variations in the value of an item over time, data distribution, sorted data ranking (high to low, worst to best). Data is represented using a collection of rectangles that extend from the top to bottom of the chart towards the values of data points. Learn more about our [column charts](chart-types-column.md).
+
+### Polar Area chart 
+
+The $PlatformShort$ Polar Area Chart or Polar Graph belongs to a group of polar charts and has a shape of a filled polygon which vertices or corners are located at the polar (angle/radius) coordinates of data points. The Polar Area Chart uses the same concepts of data plotting as the Scatter Chart but wraps data points around a circle rather than stretching them along a horizontal line. Like with other series types, multiple Polar Area Charts can be plotted in the same data chart and they can be overlaid on each other to show differences and similarities between data sets. Learn more about our [polar charts](chart-types-polar.md).
+
+### Scatter chart 
+
+The $PlatformShort$ Scatter Chart, or Scatter Graph, is used to show the relationship between two values using a Cartesian (X, Y) coordinate system to plot data. Each data point is rendered as the intersecting point of the data value on the X and Y Axis. Scatter charts draw attention to uneven intervals or clusters of data. They can highlight the deviation of collected data from predicted results and they are often used to plot scientific and statistical data. The $PlatformShort$ Scatter chart organizes and plots data chronologically (even if the data is not in chronological order before binding) on X-Axis and Y-Axis. Learn more about our [scatter charts](chart-types-scatter.md).
+
```

Comment:
We can use this cool example that we got here: 
https://www.infragistics.com/products/ignite-ui-angular/angular/components/data-chart-type-scatter-area-series

or if we want to use a more standard Scatter plot like this one:
![Birth rate v Death rate in Africa and Europe](https://user-images.githubusercontent.com/57216334/106144502-f0e04280-617c-11eb-8541-3487cb1031aa.png)

Reply:
I'd go for a more standard plot, the other one you shared is too specific

Reply:
Here is the data:

[birth-rate-vs-death-rate.xlsx](https://github.com/IgniteUI/igniteui-xplat-docs/files/5888193/birth-rate-vs-death-rate.xlsx)

Reply:
ok, we are creating this sample